wgsl/extract-grammar.py

```diff
@@ -383,8 +635,8 @@ class ScanResult(dict):
          The name is taken from the "heading" attriute of the <div> element.
     """
     def __init__(self):
-        self['rule'] = dict()
-        self['example'] = dict()
+        for subscanner in Scanner.__subclasses__():
```

Comment:
nit: I would probably find it easier to read if this was kept explicit.
It also avoids accidentally including something if we add a different scanner for an entirely different purpose.